.goreleaser.yml

```diff
@@ -11,30 +11,61 @@ builds:
   - amd64
   - arm64
   env:
-  # Compiles against the MacOS FSEvents system libraries.
+  # clang is a cross-compiler by default, so the same CC/CXX works
+  # for arm64 (M1) & amd64 (Intel) builds
   - CGO_ENABLED=1
-  - CC=/osxcross/target/bin/x86_64h-apple-darwin20.2-clang
-  - CXX=/osxcross/target/bin/x86_64h-apple-darwin20.2-clang++
+  - CC=o64h-clang
+  - CXX=o64h-clang++
```

Comment:
The version changed to `darwin20.4` but these are generic symlinks (and exist in `PATH` so don't need to be qualified), which should make future base image updates less painful